CMakeLists.txt

```diff
@@ -132,7 +132,7 @@ include(external/warpctc)   # download, build, install warpctc
 include(external/any)       # download libn::any
 include(external/eigen)     # download eigen3
 include(external/pybind11)  # download pybind11
-include(external/nccl)
+#include(external/nccl)
```

Comment:
Why removing from cmake? Is that the base Docker image have already contained nccl?

Reply:
nccl2 is now in cuda images. It is on a release schedule as part of deep learning software and no longer open sourced as opposite to nccl 1.3 on github.
Using both version in the project is both confusing and causing some error, we suggest to use nccl2 in the image as similar to cudnn moving forward

Reply:
I see, need a way to put nccl2 into https://github.com/PaddlePaddle/buildtools in order to make the CI pass.